network-api/networkapi/mozfest/models.py

```diff
@@ -199,6 +199,7 @@ class MozfestHomepage(MozfestPrimaryPage):
         "are no slides present.",
         blank=True,
         null=True,
+        use_json_field=True,
```

Comment:
I am surprised there are not more changes like this? Have all the other `StreamField`s already been updated to JSON?

Reply:
Yea we caught most of them in the [previous upgrade](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/9904), this was one that got missed